bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -167,21 +167,27 @@ fn main() -> Result<()> {
     let function_name =
         env::var("AWS_LAMBDA_FUNCTION_NAME").expect("could not read AWS_LAMBDA_FUNCTION_NAME");
     let function_arn = build_function_arn(&r.account_id, &region, &function_name);
-
-    let logs_agent = LogsAgent::run(&function_arn, Arc::clone(&config));
-    let event_bus = EventBus::run();
     let metadata_hash = hash_map::HashMap::from([(
         lambda::tags::FUNCTION_ARN_KEY.to_string(),
         function_arn.clone(),
     )]);
-    let tags_provider =
-        provider::Provider::new(Arc::clone(&config), "lambda".to_string(), &metadata_hash);
+    let tags_provider = Arc::new(provider::Provider::new(
+        Arc::clone(&config),
+        "lambda".to_string(),
```

Comment:
If this keeps repeating, we might have to set it to a constant, I think I've seen `"lambda".to_string()` a couple times now

Reply:
It should be a constant though, at least then the compiler will intern it

Reply:
Ok! I think I saw it in logs too, but maybe they are there for different purposes!